src/components/ModalDelete/ModalDeleteComponent.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteNoteAction } from 'components/BottomBar/BottomBarActions';
+import { makeStyles } from '@material-ui/core';
+import { colors } from '../../colors';
+
+const useStyles = makeStyles(() => ({
+  dialog: {
+    color: 'white',
+    backgroundColor: colors.mainGray,
+  },
+  title: {
+    color: colors.white87,
+  },
+}));
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+export default function ModalDeleteComponent({ handleClose, open }) {
+  const selectedNote = useSelector(state => state.notesMindMap.selectedNote);
+  const dispatch = useDispatch();
+  const handleDeleteButtonClick = () => {
+    dispatch(deleteNoteAction(selectedNote));
+  };
+  const classes = useStyles();
+  return (
+    <div>
+      <Dialog
+        open={open}
+        TransitionComponent={Transition}
+        keepMounted
+        onClose={handleClose}
+        aria-labelledby="alert-dialog-slide-title"
+        aria-describedby="alert-dialog-slide-description">
+        <DialogTitle className={classes.dialog} id="alert-dialog-slide-title">
```

Comment:
The `id` should be unique across the whole html document.
"alert-dialog-slide-title" doesn't sound like a unique name: if we add more dialogs they might also have the same id, copied from the Material UI docs :)
Since you're using `keepMounted` prop, this component will always be kept in the DOM.
So we have a real chance of having two elements with the same `id` in the DOM, preventing the `aria-labelledby` from finding the correct element in the DOM (it will take the first element it found by the id).

Let's give it a more unique id, like `"delete-note-confirmation-dialog-title"`.
This way if we add another dialogue in the future, it will most probably have a different `id`.

Same for the `alert-dialog-slide-description` id.